components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/store-api.yaml

```diff
@@ -4344,8 +4344,27 @@ definitions:
         type: array
         description: The subscription tiers selected for the particular API
         items:
-          type: string
-        example: ["Unlimited"]
+          properties:
+            tierName:
+              type: string
+              example: Gold
+            tierPlan:
+              type: string
+              example: COMMERCIAL
+            monetizationAttributes:
+              properties:
+                fixedPrice:
+                  type: string
+                  example: "10"
+                pricePerRequest:
+                  type: string
+                  example: "1"
+                currencyType:
+                  type: string
+                  example: "LKR"
```

Comment:
Shall we put USD

Reply:
https://github.com/wso2/carbon-apimgt/pull/6985/commits/d0f128d22d1412e1896864bde29ce15983ba11f0